files/KoreBuild/KoreBuild.sh

```diff
@@ -71,10 +72,11 @@ install_tools() {
 
     # Instructs MSBuild where to find .NET Framework reference assemblies
     export ReferenceAssemblyRoot="$tools_home/netfx/$netfx_version"
-    
+
     # Call "sync" between "chmod" and execution to prevent "text file busy" error in Docker (aufs)
     chmod +x "$__korebuild_dir/scripts/get-netfx.sh"; sync
-    "$__korebuild_dir/scripts/get-netfx.sh" $verbose_flag $netfx_version "$tools_source" "$ReferenceAssemblyRoot" \
+    # we don't include netfx in the BuildTools artifacts currently, it ends up on the blob store through other means, so we'll only look for it in the default_tools_source
+    "$__korebuild_dir/scripts/get-netfx.sh" $verbose_flag $netfx_version "$default_tools_source" "$ReferenceAssemblyRoot" \
```

Comment:
Why this change? This should be reverted as it makes it impossible to override `--tools-source` for the netfx asset.

Reply:
Like the comment says, overriding --tools-source for the netfx asset doesn't work right now. Currently the buildtools artifact does not contain netfx, it gets added to the blob store by some other process. If we want to fix that I feel it's better dealt with in another issue/PR.

Reply:
Ok, we'll track that here: https://github.com/aspnet/BuildTools/issues/399